CHANGELOG.md

```diff
@@ -86,6 +86,8 @@
   present only when the argument is of type that has a predefined set of values.
 - [Separate component browser navigator sections for modules imported from
   different namespaces][4044]
+- [Added dynamic suggestions for function arguments][4072]. Dropdowns will now
+  contain suggestions which are based on evaluated data.
```

Comment:
This is user-facing logs. The sentence "Added dynamic suggestions for function arguments" is not understandable by people not being developers of this codebase. Please describe that in a descriptive, nice way. Just like if you were writing a blog post about it, please :)